app/components/search/SearchResults/SearchResults.module.scss

```diff
@@ -138,6 +138,9 @@
 .sideLinks {
   flex-shrink: 0;
   width: auto;
+  @media screen and (min-width: $min-tablet-p) {
```

Comment:
The translation of "send a text message" link was rather lengthy in Spanish. I set this max-width to force the link to wrap as it was extending behind the search results map. I spent about an hour fooling around with `flex-wrap/grow/shrink`, `overflow`, `white-space`, `word-break` but with no success. Thus, I just decided to set the max-width. I'm open to better solutions if anyone has any, though!